commands/download.go

```diff
@@ -7,8 +7,9 @@ import (
 	"github.com/jfrog/jfrog-client-go/utils/log"
 )
 
+//nolint: unparam, lll
```

Comment:
Should we add a todo here? It seems better to fix the problem than ignore it.

Reply:
I though you'd be able to fix this in your PR #7

Reply:
yes, I'll do it. I have a bunch of refactoring to do anyways.